scripts/defenseclaw-rescue.ps1

```
@@ -0,0 +1,1076 @@
+    }
+    if ($null -ne $item.PSObject.Methods["SetAccessControl"]) {
+        $item.SetAccessControl($security)
+    } else {
+        [IO.FileSystemAclExtensions]::SetAccessControl(
+            $item,
+            [Security.AccessControl.DirectorySecurity]$security
+        )
+    }
+}
+
+function New-PrivateStageRoot {
```

PSScriptAnalyzer (1.25.0)

[warning] Function 'New-PrivateStageRoot' has verb that could change system state. Therefore, the function has to support 'ShouldProcess'.

(PSUseShouldProcessForStateChangingFunctions)